### PR TITLE
ci: skip cdc ci for dm's pr

### PR DESCRIPTION
**Pull request by purelind**
* skip cdc integration test for dm's pr

### CODE REVIEW

**Comment by purelind** [comment]
replay cdc_ghpr_integration_test example:
1. not skip   https://ci2.pingcap.net/job/cdc_ghpr_integration_test/9321/console
2. skip  https://ci2.pingcap.net/job/cdc_ghpr_integration_test/9322/console

File: jenkins/pipelines/ci/ticdc/cdc_ghpr_integration_test.groovy [hunk 1]
```diff
@@ -94,6 +94,75 @@ println "BUILD_NODE_NAME=${GO_BUILD_SLAVE}"
...
+
+
+def check_pr_diff_files() {
+    def files = list_pr_diff_files()
+    def pattern = /^dm\/.*$/
+    // if all diff files start with dm/, skip cdc integration test
+    def matched = pattern_match_all_files(pattern, files)
+    if (matched) {
+        echo "matched, all diff files full path start with dm/, current pr is dm's pr(not related to ticdc), skip cdc integration test"
+        throw new Exception("skip cdc integration test for dm's pr")
```
Comment (Ehco1996) [inline jenkins/pipelines/ci/ticdc/cdc_ghpr_integration_test.groovy:145]: why use a exception as return value? can we just retun bool (need run or not)?

Comment (purelind) [reply jenkins/pipelines/ci/ticdc/cdc_ghpr_integration_test.groovy]: At first, I wanted to put it in the integration_test_common.groovy， we need to use exceptions to handle that condition. Currently, I will modify it to return bool directly.

**Review by Ehco1996 — APPROVED** [review]

**Review by dianqihanwangzi — APPROVED** [review]
lgtm

**Pull request merged**